### PR TITLE
Fix event type

### PR DESCRIPTION
**Pull request by isuruf**
I added `issue` after the other 4 and forgot to test. Sorry about that.

### CODE REVIEW

File: conda_smithy/ci_register.py [hunk 1]
```diff
@@ -379,7 +379,7 @@ def add_conda_forge_webservice_hooks(user, repo):
             "http://conda-forge.herokuapp.com/conda-forge-command/hook",
             [
                 "pull_request_review", "pull_request",
-                "pull_request_review_comment", "issue_comment", "issue",
+                "pull_request_review_comment", "issue_comment", "issues",
```
Comment (jakirkham) [inline conda_smithy/ci_register.py]: Honestly we might want to drop the [`issues` event]( https://developer.github.com/v3/activity/events/types/#issuesevent ). It seems more focused on metadata about the issue than anything that would be considered an actionable comment.

Comment (isuruf) [reply conda_smithy/ci_register.py]: There is `issue[title]` and `issue[body]` there. Things like rechecking the team can be commands given in an issue.

Comment (jakirkham) [reply conda_smithy/ci_register.py]: So you are thinking of issue ( https://github.com/conda-forge/conda-forge-webservices/issues/104 ).

Comment (isuruf) [reply conda_smithy/ci_register.py]: Yes, exactly

Comment (jakirkham) [reply conda_smithy/ci_register.py]: We should definitely discuss that use case more. That said, I'm ok with going ahead with this change. There may be some other things to explore like assignees and review requests.

**Pull request merged**